docs/user-guide/interfaces/cli/developing-rugs.md

```diff
@@ -172,6 +166,18 @@ be faster.
 As you can see from the last line of output, all of the test scenarios
 passed.
 
+To run a specific named test:
+
+```console
+$ rug test "Whatever Test Secanrio"
+```
+
+To run all scenarios from a .rt file:
+
+```console
+$ rug test MyRugTestFilename
+```
+
```

Comment:
Yes, it does. Sorry for the confusion. Just make sure not to mention the old .rt files.